homework/index.js

```diff
@@ -15,32 +38,127 @@
     xhr.onerror = () => cb(new Error('Network request failed'));
     xhr.send();
   }
-
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
+  // *********************************************************
+  function ShowContributions(urlText) {
```

Comment:
Function names should be camelCase (start with a lowercase letter), unless it is a constructor function (which in this case, it is not).

Reply:
yes , that is my typing mistake while I was renaming almost everything.